teuthology/orchestra/remote.py

```diff
@@ -276,11 +277,20 @@ def write_file(self, path, data, sudo=False, mode=None, owner=None,
         :param owner:   set file owner if provided
         :param mkdir:   preliminary create the file directory, defaults False
         :param append:  append data to the file, defaults False
+        :param bs:      block size
+        :param offset:  number of bytes to seek to in file
+        :param sync:    sync file
```

Comment:
Add default value description.

Reply:
Yeah... here is another point to elaborate, so user understand how to use _sync_ argument.